release-tools/stage-release.sh

```diff
@@ -1,11 +1,13 @@
-#! /bin/bash -e
+#! /usr/bin/env bash
 # Copyright 2020-2023 The OpenSSL Project Authors. All Rights Reserved.
 #
 # Licensed under the Apache License 2.0 (the "License").  You may not use
 # this file except in compliance with the License.  You can obtain a copy
 # in the file LICENSE in the source distribution or at
 # https://www.openssl.org/source/license.html
 
+set -e
+
```

Comment:
Side note: the move of `-e` was a bit surprising...  `/usr/bin/env bash -e` is as perfectly valid, and I find it more visible.  But, I'm ok with this, still approving this PR.

Reply:
Is it? UNIX shebangs are weird... some UNIX platforms allow an arbitrary number of arguments on the shebang line, but many (including Linux) are weird and make everything a single argument, basically only splitting on the first space. So `#!/bin/foo alpha beta gamma` is equivalent to `/bin/foo "alpha beta gamma"`.

Reply:
FWIW: personally, I prefer the more verbose line

```
set -o errexit
```

Reply:
Don't know whether that's a bashism though.

Reply:
The original and the best: `set -e` please.

Reply:
... says the former participant of the IOCCC 😀